web/blog/vast-v3.0/index.md

```diff
@@ -76,7 +76,7 @@ do for a long time. Here's a summary:
    previously supported `#type` meta extractor.
 
 4. We removed the `#field` meta extractor. That is, queries of the form `#field
-   == "some.field.name"` no longer work. Use `some.field.name != nil` to check
+   == "some.field.name"` no longer work. Use `some.field.name != null` to check
    for field existence moving forward.
 
 5. We renamed the boolean literal values `T` and `F` to `true` and `false`,
```

Comment:
Please add to this that we renamed the literal for null values from `nil` to `null`.